frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { createSelector, createFeatureSelector } from '@ngrx/store';
+import { FeatureFlagState } from './feature-flags.model';
+import { selectRouterState } from '../../core.state';
+import { selectAll } from './feature-flags.reducer';
+
+export const selectFeatureFlagsState = createFeatureSelector<FeatureFlagState>('featureFlags');
+
+export const selectAllFeatureFlags = createSelector(selectFeatureFlagsState, selectAll);
+
+export const selectIsLoadingFeatureFlags = createSelector(
+  selectFeatureFlagsState,
+  (state) => state.isLoadingFeatureFlags
+);
+
```

Comment:
export const selectSelectedFeatureFlag = createSelector(
  selectRouterState,
  selectFeatureFlagsState,
  ({ state: { params } }, featureFlagState) => featureFlagState.entities[params.flagId]);

Reply:
is exactly the same as what is there now, never a need a ternary for this kind of logic to return undefined